ConcordiumWallet/Resources/BakerRegisterOnboarding/baker_intro_flow_en_1.html

```diff
@@ -9,7 +9,7 @@
 
 <body>
     <p>
-    A baker is a node that participates in the network by baking (creating) new blocks that are added to the chain. Each baker has a set of cryptographic keys called baker keys that the node needs to bake blocks. You generate the baker keys in the Mobile Wallet when you add a baker account. Once the baker node has been restarted with the baker keys, it will start baking two epochs after the transaction has been approved.
+    A validator is a node that participates in the network by producing (creating) new blocks that are added to the chain. Each validator has a set of cryptographic keys called validator keys that the node needs to produce blocks. You generate the validator keys in the Mobile Wallet when you add a validator account. Once the validator node has been restarted with the validator keys, it will start producing two epochs after the transaction has been approved.
```

Comment:
```suggestion
    A validator is a node that participates in the network by producing (creating) new blocks that are added to the chain. Each validator has a set of cryptographic keys called validator keys that the node needs to produce blocks. You generate the validator keys in the Mobile Wallet when you add a validator account. If the validator node has been restarted with the validator keys, it will start producing blocks two epochs after the transaction was added to the chain.
```

Saying "added" to intentionally avoid phrases like "included in a block that got finalized blah blah".